HoloPipelines/models/dense_vnet_abdominal_ct/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#Download base image ubuntu 18.04
+FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+
+
+# Set variables
+ARG APP_DIR=/usr/src/app
+ARG PORT=5000
+
+# Update Software repository, install package and clean the package 
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends python3-pip python3-dev build-essential nano &&\
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+
+
+# Copy the flask script into the container and set the working directory
+RUN mkdir -p ${APP_DIR}
+WORKDIR ${APP_DIR}
+COPY server.py test.py ./
+
+# Copy the requirements file in to image for pip download
+COPY requirements.txt ./
+
+# Run install tensorflow, niftynet, flask
+RUN pip3 install --upgrade pip setuptools
+RUN pip3 install wheel
+RUN pip3 install -r requirements.txt
```

Comment:
I spotted this error in the stdout, not sure if it is actually a problem? Installation continued afterwards:
```shell
ERROR: niftynet 0.5.0 has requirement numpy<=1.14.5,>=1.13.3, but you'll have numpy 1.17.0 which is incompatible.
```

Reply:
I think this ERROR is fine, niftynet will later reinstall the numpy to a version that fits the requirement